teacher.rb

```diff
@@ -0,0 +1,15 @@
+# Create class Teacher with the following:
+# Inherits from Person.
+# Constructor extends parent's constructor by adding @specialization and a parameter for it.
+# Override can_use_services? so it always returns true.
+
+class Teacher < Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(name, parent_permission, age)
+    @specialization = specialization
```

Comment:
- [ ] Please add a parameter for `@specialization` in the initialize method like that:
```ruby
def initialize(age, name = 'Unknown', parent_permission: true, specialization)
```